scripts/Creative.zs

```diff
@@ -218,7 +273,7 @@ var twilightForestMasterTrophy = <simple_trophies:trophy>.withTag({
 	[hvSolar, ultProvider, ultProvider, anglesite, anglesite, anglesite, ultProvider, ultProvider, hvSolar], 
 	[hvSolar, ultProvider, <ic2:te:75>, <ic2:te:22>, <nuclearcraft:fusion_core>, <ic2:te:22>, <ic2:te:75>, ultProvider, hvSolar], 
 	[hvSolar, anglesite, <ic2:te:22>, ultCell, ultCatalyst, ultCell, <ic2:te:22>, anglesite, hvSolar], 
```

Comment:
Is this changing the recipe from requiring the full generator to only half?

Reply:
I suggest to add Rainbow Stone instead of rainbow generator. Because it require working Rainbow Generator what harder to make than just generator itself. Through, to make one stone you not need to bother with fully automatic working generator.
https://i.imgur.com/aTe6RbW.png